types/sentry.d.ts

```diff
@@ -1,6 +1,7 @@
 import { Options as WebpackOptions } from 'webpack'
 import { Options as SentryOptions } from '@sentry/types'
 import { SentryCliPluginOptions } from '@sentry/webpack-plugin'
+import { ParseRequestOptions } from '@sentry/node/dist/handlers'
```

Comment:
Relying on the internal directory structure is error-prone. You can do:
```suggestion
import { Handlers } from '@sentry/node'
```
and then use `Handlers.ParseRequestOptions`.

Reply:
BTW. Created an issue and PR at Sentry to expose the final type so that we don't have to duplicate `flushTimeout`: https://github.com/getsentry/sentry-javascript/issues/2994

Reply:
The new type is now integrated and sentry dependency is updated on master. You can rebase and simplify this now.

Reply:
I've pushed the change.